installation/sql/postgresql/base.sql

```diff
@@ -380,6 +380,7 @@ INSERT INTO "#__extensions" ("package_id", "name", "type", "element", "folder",
 (0, 'plg_task_sessiongc', 'plugin', 'sessiongc', 'task', 0, 1, 1, 0, 1, '', '{}', '', 7, 0),
 (0, 'plg_task_sitestatus', 'plugin', 'sitestatus', 'task', 0, 1, 1, 0, 1, '', '{}', '', 8, 0),
 (0, 'plg_task_updatenotification', 'plugin', 'updatenotification', 'task', 0, 1, 1, 0, 1, '', '{}', '', 9, 0),
+(0, 'plg_task_stats', 'plugin', 'stats', 'task', 0, 1, 1, 0, 1, '', '{}', '', 10, 0),
```

Comment:
```suggestion
(0, 'plg_task_sitestatus', 'plugin', 'sitestatus', 'task', 0, 1, 1, 0, 1, '', '{}', '', 8, 0),
(0, 'plg_task_stats', 'plugin', 'stats', 'task', 0, 1, 1, 0, 1, '', '{}', '', 9, 0),
(0, 'plg_task_updatenotification', 'plugin', 'updatenotification', 'task', 0, 1, 1, 0, 1, '', '{}', '', 10, 0),
```
Please alpha order the plugins of the same folder.